src/components/Form.js

```diff
@@ -307,35 +308,52 @@ class Form extends React.Component {
 
     render() {
         const scrollViewContent = safeAreaPaddingBottomStyle => (
-            <FormSubmit style={StyleSheet.flatten([this.props.style, safeAreaPaddingBottomStyle])} onSubmit={this.submit}>
-                {this.childrenWrapperWithProps(_.isFunction(this.props.children) ? this.props.children({inputValues: this.state.inputValues}) : this.props.children)}
-                {this.props.isSubmitButtonVisible && (
-                <FormAlertWithSubmitButton
-                    buttonText={this.props.submitButtonText}
-                    isAlertVisible={_.size(this.state.errors) > 0 || Boolean(this.getErrorMessage()) || !_.isEmpty(this.props.formState.errorFields)}
-                    isLoading={this.props.formState.isLoading}
-                    message={_.isEmpty(this.props.formState.errorFields) ? this.getErrorMessage() : null}
-                    onSubmit={this.submit}
-                    onFixTheErrorsLinkPressed={() => {
-                        const errors = !_.isEmpty(this.state.errors) ? this.state.errors : this.props.formState.errorFields;
-                        const focusKey = _.find(_.keys(this.inputRefs), key => _.keys(errors).includes(key));
-                        const focusInput = this.inputRefs[focusKey];
-                        if (focusInput.focus && typeof focusInput.focus === 'function') {
-                            focusInput.focus();
-                        }
-
-                        // We subtract 10 to scroll slightly above the input
-                        if (focusInput.measureLayout && typeof focusInput.measureLayout === 'function') {
-                            focusInput.measureLayout(this.formRef.current, (x, y) => this.formRef.current.scrollTo({y: y - 10, animated: false}));
-                        }
-                    }}
-                    containerStyles={[styles.mh0, styles.mt5, styles.flex1]}
-                    enabledWhenOffline={this.props.enabledWhenOffline}
-                    isSubmitActionDangerous={this.props.isSubmitActionDangerous}
-                    disablePressOnEnter
-                />
-                )}
-            </FormSubmit>
+            <View ref={this.formContentRef}>
```

Comment:
Please note that this big diff is more readable if you read stuff commit-by-commit!

Reply:
@parasharrajat I didn't know that exists.

Would you expand on _why_ would I use that, if that's undocumented? I assume that it has no guarantees, and can disappear in a patch release, like all undocumented things.

Reply:
Have you tested this `innerViewRef` in context of the problem I'm fixing? It doesn't seem to be working.

Reply:
I still removed the extra view, to be double sure that we'll not introduce styling issues. At the same time, I used the inner view of `FormSubmit`, instead of `innerViewRef` from `ScrollView`, as that didn't seem to fix the problem I'm fixing.